unime/src/i18n/en/index.ts

```diff
@@ -254,6 +254,9 @@ const en = {
   },
   TIMELINE: {
     EMPTY: 'No activity yet.',
+    CREDENTIALS_ADDED: 'Received credentials from',
+    CREDENTIALS_SHARED: 'Shared credentials with',
```

Comment:
Isn't it "Shared **data** with" instead of "Shared **credentials** with"?

Reply:
Same for "Received **data** from", right?

Reply:
Yes but I discussed with Andrew and he seems to prefer credentials as well, however if more than just credentials will be shared we can use data. Its more used as an umbrella term which could include more.